build/Dockerfile

```diff
@@ -4,6 +4,14 @@ ENV OPERATOR=/usr/local/bin/mongodb-kubernetes-operator \
     USER_UID=1001 \
     USER_NAME=mongodb-kubernetes-operator
 
+RUN apt-get update  && \
+    apt-get install -y curl
+
+ENV manifest_version=4.4
+RUN mkdir -p /content/ \
+        && chmod -R +r /content/ \
+        && curl --fail --retry 3 -o /usr/local/version_manifest.json "https://opsmanager.mongodb.com/static/version_manifest/${manifest_version}.json"
```

Comment:
small update to add the version manifest to the "faster" local dev build using `operator-sdk build`